otelib/backends/python/transformation.py

```diff
@@ -0,0 +1,19 @@
+"""Transformation strategy."""
+from typing import TYPE_CHECKING
+
+from oteapi.models import TransformationConfig
+
+from .base import BasePythonStrategy
+
+if TYPE_CHECKING:  # pragma: no cover
+    from typing import Optional
+
+
+class Transformation(BasePythonStrategy):
```

Comment:
I was under the impression that these are not yet implemented in core?

Reply:
They are, but let's not do it anyway, since we are looking to streamline this strategy. We should instead create an issue for it to inspect it after "fixing" the intended usage of the transformation strategy in core.